activiti-cloud-starter-audit-mongo/src/main/resources/META-INF/spring.factories

```diff
@@ -0,0 +1,2 @@
+org.springframework.boot.autoconfigure.EnableAutoConfiguration=\
+    org.activiti.cloud.starter.audit.configuration.ActivitiAuditMongoAutoConfiguration
```

Comment:
I was reading about SpringBoot auto configuration and it seems that we are doing it wrong:

> Auto-configurations must be loaded that way only. Make sure that they are defined in a specific package space and that, in particular, they are never the target of component scanning.

https://docs.spring.io/spring-boot/docs/current/reference/html/boot-features-developing-auto-configuration.html#boot-features-locating-auto-configuration-candidates